pkg/virt-controller/services/virtiofs.go

```diff
@@ -15,6 +15,32 @@ import (
 	"kubevirt.io/kubevirt/pkg/virtiofs"
 )
 
+// RequireAnyVirtiofsPrivilegedContainer Returns true if any of the returns true if any
+// of the filesystems requires a container of virtiofs to run as root
+func RequireAnyVirtiofsPrivilegedContainer(spec *v1.VirtualMachineInstanceSpec) bool {
```

Comment:
Nit: Can we move this to where we use it?  (Reasoning we are working on dividing packages for different groups. The webhook will be part of sig-api and this function would be part other sig. Meaning the sig-scale would not be able to get the changes in on their own)

Reply:
Sure, by doing that, I could also change the functional test (in the last commit) to be just a unit test for the admitter.

But, how can I do it without "leaking" the knowledge how which volumes require root, and which do not require root?
I think this info should, at least, be inside the virtiofsd pakage, would it be enough if I move it to `pkg/virtiofs/virtiofs.go`?

Reply:
@germag Thanks, now I see the point.
We can expose the `isConfig` or move this as whole. I leave it for you